nexus/tests/integration_tests/instances.rs

```diff
@@ -530,6 +535,313 @@ async fn test_instances_invalid_creation_returns_bad_request(
         .starts_with("unable to parse body: invalid value: integer `-3`"));
 }
 
+// Test creating two new interfaces for an instance, at creation time.
+#[nexus_test]
+async fn test_instance_with_new_custom_network_interfaces(
```

Comment:
Are there tests that:

- specify a specific IP address, with no conflict
- specify a specific IP address, with a conflict (i.e., conflicts with the first one)

Reply:
Yes, in the process of adding these to test the full saga unwind.